pkg/kubelet/kubelet_node_status.go

```diff
@@ -463,7 +463,7 @@ func (kl *Kubelet) tryUpdateNodeStatus(tryNumber int) error {
 	// If update finishes successfully, mark the volumeInUse as reportedInUse to indicate
 	// those volumes are already updated in the node's status
 	kl.volumeManager.MarkVolumesAsReportedInUse(updatedNode.Status.VolumesInUse)
-	return nil
+	return kl.containerManager.UpdateNodeAllocatableCgroups(node.Status.Capacity)
```

Comment:
Also, is there a reason why you use `node` and not `updatedNode`? If `node` is correct, I think it makes more sense to put this before the `nodeutil.PatchNodeStatus`.